manifests/init.pp

```diff
@@ -31,7 +31,7 @@
 
   rgbank::load { $load_component:
     balancermembers => $web_https,
-    port            => $serve_port,
+    port            => $listen_port,
```

Comment:
This isn't going to work... we need to have the ability to split these ports for the single host scenarios.